### PR TITLE
Minor changes to exceptional ops

### PR DESCRIPTION
**Pull request by jhdalek55**
Can we get some eyes on the section about aftermarket ECUs? When I was reviewing this for tone, I found the last sentence in that paragraph was incomplete or just missing a few critical word or two. I attempted to fix it, but I don't know if more needed to be done.

Lois

### CODE REVIEW

File: exceptional_operations.md [hunk 1]
```diff
@@ -39,30 +39,30 @@ where attackers compromise only the Director repository.  However, there are two
...
 
 ### Aftermarket ECUs
 
-A slightly more difficult use case to deal with are aftermarket ECUs--for example, 3rd-party replacement parts, or add-on ECUs that add functionality for commercial fleet management. One approach is to work with the ECU manufacturer, and treat them like any other tier-1. (The addition of the aftermarket ECU would be managed in one of the ways recommended in the previous section.) However, this is likely not economically feasible in many/most cases. The easiest alternative is to simply exclude the aftermarket ECU from receiving OTA updates.
+A slightly more difficult use case to deal with are aftermarket ECUs--for example, 3rd-party replacement parts, or add-on ECUs that add functionality for commercial fleet management. One approach is to work with the ECU manufacturer and treat them like any other tier-1. (The addition of the aftermarket ECU would be managed in one of the ways recommended in the previous section.) However, this is likely not economically feasible in many/most cases. The easiest alternative is to simply exclude the aftermarket ECU from receiving OTA updates.
```
Comment (patrickvacek) [inline exceptional_operations.md]: "many/most" sounds odd; maybe just simplify to "many"?

File: exceptional_operations.md [hunk 2]
```diff
@@ -39,30 +39,30 @@ where attackers compromise only the Director repository.  However, there are two
...
 
-Many aftermarket ECUs for fleet management, monitoring, or  have their own independent internet connection, and thus do not need to be integrated into the OEM's update system at all.
+Many aftermarket ECUs for fleet management or monitoring have their own independent internet connection, and thus do not need to be integrated into the OEM's update system at all.
```
Comment (patrickvacek) [inline exceptional_operations.md]:
```suggestion
Some aftermarket ECUs, such as those designed for fleet management or monitoring, may have their own independent internet connection, and thus do not need to be integrated into the OEM's update system at all.
```
I think this flows a bit better. As far as the mysterious missing third item in the list, I think it's fine as it is. That bit was added by @tkfu, so if you remember whatever you might've been thinking of when you wrote that, feel free to add it back in!

File: exceptional_operations.md (hunk 1 above)
Comment (jhdalek55) [reply exceptional_operations.md]: Agreed

**Comment by jhdalek55** [comment]
Can we maybe get one other review?

File: exceptional_operations.md (hunk 2 above)
Comment (tkfu) [reply exceptional_operations.md]: Whatever I was thinking of has been lost to the mists of time, but I agree that this flows well and gets the point across.

**Pull request merged**